turbopack/crates/turbopack-core/src/module_graph/mod.rs

```diff
@@ -47,17 +47,24 @@ pub use self::module_batches::BatchingConfig;
 )]
 pub struct GraphNodeIndex {
     #[turbo_tasks(trace_ignore)]
-    graph_idx: usize,
+    graph_idx: u32,
     #[turbo_tasks(trace_ignore)]
     node_idx: NodeIndex,
 }
+impl GraphNodeIndex {
+    #[inline]
+    fn graph_idx(&self) -> usize {
```

Comment:
```suggestion
    const fn graph_idx(&self) -> usize {
```

I guess this is the best pattern here.

Reply:
Done.